### ISSUE TITLE
Environment variables from .secret.local not loading when using functions emulator

### ISSUE THREAD

**Issue by 8ctavio** [firebase/firebase-tools#5648]
### [REQUIRED] Environment info

**firebase-tools: v11.25.2**

**Platform: Windows 11**

**Node.js: v18.15.0**

### [REQUIRED] Test case

- Create an empty directory. `cd` into it.
- Run `firebase init`. Only select cloud functions and emulators features, and functions emulator. Choose a project. Install dependencies and emulator.
- Create `.env.local` and `.secret.local` files inside the `./functions` directory.

### [REQUIRED] Steps to reproduce

Start the emulator with `firebase emulators:start`.

### [REQUIRED] Expected behavior

Environment variables from both `.env.local` and `.secret.local` are loaded. 

### [REQUIRED] Actual behavior

Only environment variables from `.env.local` are loaded. In `firebase-debug.log` it gets logged:

```console
[info] i  functions: Loaded environment variables from .env.local. 
```

**Comment by aalej** [comment]
Hi @8ctavio. From what I understand, when running the Functions emulator, you are able to load the contents of the `.env.local` file but not the `.secret.local` file. Just to verify, could you try:

1. In your `functions/index.js` file, add a code snippet similar to:
```
exports.helloWorldTest = functions
 .runWith({ secrets: ["TEST_SECRET_KEY"] })
 .https.onRequest((request, response) => {
   functions.logger.info("Hello logs!", { structuredData: true });
   functions.logger.info(`SECRET: ${process.env.TEST_SECRET_KEY}`);
   response.send("Hello from Firebase!");
 });
```
2. In your `functions/.secret.local`, add:
```
TEST_SECRET_KEY=test_secret_value
```
3. Run `firebase emulators:start`

After running the emulator, try calling the http function locally. Are you able to see the value of the secret in the logs? The logs should show something similar to:
```
i  functions: Beginning execution of "us-central1-helloWorldTest"
>  {"structuredData":true,"severity":"INFO","message":"Hello logs!"}
>  {"severity":"INFO","message":"SECRET: test_secret_value"}
i  functions: Finished "us-central1-helloWorldTest" in 9.284151ms
```

**Comment by 8ctavio** [comment]
Hi @aalej,

I tried that and it works now. I first had the problem in a bigger project with a slightly different environment, and when testing, I couldn't access secret keys inside functions. When I then built and tested the demo with the new environment, I must only have checked for the logs to state that environment variables from `.secret.local` were loaded, since I think that used to get logged as well; in the logs I'm still seeing that only environment variables from `.env.local` are loaded. However, now I'm able to use environment variables from `.secret.local` inside functions, so they're being loaded👍. Sorry for the inconvenience.

Thank you.
